ceph-ansible-pull-requests/config/definitions/ceph-ansible-pull-requests.yml

```diff
@@ -0,0 +1,72 @@
+- scm:
+    name: ceph-ansible
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph-ansible.git
+          branches:
+            - ${sha1}
+          refspec: +refs/pull/*:refs/remotes/origin/pr/*
+          browser: auto
+          timeout: 20
+          skip-tag: true
+          wipe-workspace: false
+          basedir: "ceph-ansible"
+
+- scm:
+    name: ceph-build
```

Comment:
@alfredodeza totally don't need it

![told-you-so](https://cloud.githubusercontent.com/assets/20618/12496740/73a5cfd2-c065-11e5-8236-b9df3fe3da34.gif)